frontend/src/pull-card/index.tsx

```diff
@@ -38,9 +39,10 @@ const SigsAndFlags = chakra(Flex, {
    }
 });
 
-export function PullCard({pull}: {pull: Pull}) {
+export const PullCard = memo(
```

Comment:
Also super nit-picky, but could we use arrow functions instead of  the following?
```
PullCard = memo( function PullCard(){
```

It feels a little repetitive- unless this is for react dev-tool analysis or something?

Reply:
> unless this is for react dev-tool analysis or something?

Yes, it's a bit unsightly, though it is only one line. Precisely. Part of the eslint-react plugin is making sure every component has a display name. In fact, this is the recommended pattern for memoized components:
https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/display-name.md